meta/Vertices.m

```diff
@@ -112,6 +151,22 @@ are scalars without (generation) indices.  This alone does not yet
 	Utils`FSPermutationSign[FindPermutation[{vectors}, sortedVectors]] * SARAH`Cp @@ sortedVectors
 ];
 
+(* sorting CXXDiagrams version of VVVV vertices *)
```

Comment:
I think it would be good to extend the documentation of this function a bit. It is quite involved. :)
Maybe one can very briefly explain what the sorting criteria is or how a sorted VVVV vertex looks like?

Reply:
I'll write it reorders the fields to canonical order as determined by `SortFieldsInCp` and give two examples: for fields with and without indices. Would that be ok?

Reply:
Yes, that would be great, thanks!